src/main/java/net/pms/encoders/VLCVideo.java

```diff
@@ -366,6 +366,10 @@ protected CodecConfig genConfig(RendererConfiguration renderer) {
 					bufSize = defaultMaxBitrates[0] / 3;
 				}
 
+				if (media.is4KVideo()) {
+					bufSize = defaultMaxBitrates[0] / 12;
+				}
```

Comment:
@Sami32 I don't know how that should help (and it didn't), as it isn't related to my CPU at all. I'm using LGA 2011-v3/X99 (i7-5820K) so there's no embedded GPU. My only GPU is a GeForce GTX 970, which apparantly doesn't support 10-bit HEVC.

Whether my GPU supports 10-bit HEVC isn't really important here, I think it'd be relatively safe to assume that transcoding 10-bit HEVC in realtime is likely to work if the GPU supports it and fail if the CPU has to be used.